torch/ao/quantization/fake_quantize.py

```diff
@@ -356,22 +356,22 @@ def forward(self, X: torch.Tensor) -> torch.Tensor:
 """
 Default fake_quant for per-channel weights.
 """
-default_embedding_fake_quant = FakeQuantize.with_args(observer=PerChannelMinMaxObserver,
+default_embedding_fake_quant = FakeQuantize.with_args(observer=MovingAveragePerChannelMinMaxObserver,
                                                       qscheme=torch.per_channel_affine_float_qparams,
                                                       dtype=torch.quint8,
                                                       quant_min=0,
                                                       quant_max=255,
                                                       ch_axis=0,
-                                                      memoryless=True)
+                                                      averaging_constant=1)
```

Comment:
Hardcoding a constant like this seems a bit arcane, especially since it is moving from a more descriptive param like "memoryless" to achieve the same behavior.  Is there a way to use a descriptively named constant, or otherwise document this intended behavior in the fakequant module?

Reply:
I can add it to the docstrings for the fake_quants.